CMakeUtilities/InstallCMakeExports.cmake

```diff
@@ -102,6 +102,8 @@ endforeach()
 # Get the absolute path out for use in the configured CMake file ImportSIMDIS_SDK.cmake
 set(SDK_LIB_CMAKE_FULLPATH "${CMAKE_INSTALL_PREFIX}/${SDK_LIB_CMAKE_PATH}")
 
+if(USING_VCPKG)
+else()
```

Comment:
Avoid:

```
if(USING_VCPKG)
else()
```

Instead just use `if(NOT USING_VCPKG)`.

I had fixed up most of these already but it looks like several were just reintroduced.